providers/databricks/src/airflow/providers/databricks/hooks/databricks.py

```diff
@@ -709,6 +768,47 @@ def update_job_permission(self, job_id: int, json: dict[str, Any]) -> dict:
         """
         return self._do_api_call(("PATCH", f"api/2.0/permissions/jobs/{job_id}"), json)
 
+    def post_sql_statement(self, json: dict[str, Any]) -> str:
+        response = self._do_api_call(("POST", "api/2.0/sql/statements"), json)
+        return response["statement_id"]
+
+    def get_sql_statement_state(self, statement_id: str) -> SQLStatementState:
+        """
+        Retrieve run state of the SQL statement.
+
+        :param statement_id: ID of the SQL statement.
+        :return: state of the SQL statement.
+        """
+        get_statement_endpoint = ("GET", f"api/2.0/sql/statements/{statement_id}")
```

Comment:
At the top of this module, we are creating several constants for endpoint prefixes like `CANCEL_RUN_ENDPOINT` and etc. Maybe, we can create "STATEMENT_ENDPOINT" and reuse it across these new methods?

Reply:
added such a constant and used it